workflows/eager.nf

```diff
@@ -133,6 +134,27 @@ workflow EAGER {
     }
 
 
+    // Sex determine
+    if ( params.sexdeterrmine_bedfile ) {
+      ch_sexdeterrmine_bed = Channel.fromPath( params.sexdeterrmine_bedfile, checkIfExists: true )
+        .collect()
+        .map {
+          file ->
+              meta = file.simpleName
+          [meta,file]
+        }
+
+    } else {
+      ch_sexdeterrmine_bed = []
+    }
```

Comment:
The sexdeterrmine bedfile should be read in as part of the `REFERENCE_INDEXING` SWF. Please see https://github.com/nf-core/eager/blob/dev/docs/development/dev_docs.md for how to add things there if they are not already present.